runtime/compiler/runtime/RelocationRuntime.hpp

```diff
@@ -303,6 +318,10 @@ class TR_RelocationRuntime {
       void setIsLoading() { _isLoading = true; }
       void resetIsLoading() { _isLoading = false; }
 
+      bool isRelocating() { return _isRelocating; }
```

Comment:
Nitpick: let's make the getter `const`. You can make `isLoading() const` too for consistency.